tests/Query.test.ts

```diff
@@ -23,16 +23,22 @@ describe('Query parsing', () => {
         'done before 2021-12-27',
         'done date is invalid',
         'done on 2021-12-27',
+        'done this week',
+        'done 2021-12-27 2021-12-29',
```

Comment:
Suggest making this change to all the ones with dates in, just so `in` is covered:

```diff
done 2021-12-27 2021-12-29
done in 2021-12-27 2021-12-29
```

Reply:
do you mean changing `done 2021-12-27 2021-12-29` to `done in 2021-12-27 2021-12-29` or just adding `done in 2021-12-27 2021-12-29`?

Reply:
Oops - I fixed the diff above so the colour coding conveys the meaning...
I mean changing the one to the other...

Sorry for being unclear!

Reply:
oh sure, this way it is aligned with the docs. I'm pushing and resolving the conversation in this PR. Pls check the changes in the new PR.